src/QROptionsTrait.php

```diff
@@ -395,6 +395,26 @@ trait QROptionsTrait{
 	 */
 	protected string $imagickFormat = 'png32';
 
+	/*
+	 * QRNetpbm settings
+	 */
+
+	/**
+	 * Use netpbm plain (ascii) output instead of binary output
+	 *
+	 * @see https://netpbm.sourceforge.net/doc/#commonoptions
+	 * @see https://github.com/chillerlan/php-qrcode/pull/323
+	 */
+	protected bool $netpbmPlain = false;
+
+	/**
+	 * Netpbm max value ( For graymap and pixmap output formats )
+	 * Should be between 0 and 65536.
+	 *
+	 * @see https://netpbm.sourceforge.net/doc/pgm.html
+	 * @see https://netpbm.sourceforge.net/doc/ppm.html
+	 */
+	protected int $netpbmMaxValue = 255;
```

Comment:
As mentioned above, this value should have a setter that just clamps the allowed values - rather than throwing - as an integer value is still something we can work with:

```php
	protected function set_netpbmMaxValue(int $netpbmMaxValue):void{
		$this->netpbmMaxValue = max(65536, min(0, $netpbmMaxValue));
	}
```

Reply:
Note that in php8.4 you can do this in a more native way:
https://www.php.net/manual/en/language.oop5.property-hooks.php

Reply:
That's correct, but v6 of this library is still intended for a minimum PHP version of 8.2. The underlying library is aware and [property hooks are already taken care of](https://github.com/chillerlan/php-settings-container/blob/main/tests/PropertyHooksTest.php).